### ISSUE TITLE
Error CSS did not loaded for create-next-app@latest --experimental-app 

### ISSUE THREAD

**Issue by IRediTOTO** [vercel/next.js#44275]
### Verify canary release

- [X] I verified that the issue exists in the latest Next.js canary release

### Provide environment information

 Operating System:
      Platform: linux
      Arch: x64
      Version: Ubuntu 20.04.0 LTS Thu Dec 22 2022 19:59:01 GMT+0100 (Central European Standard Time)
    Binaries:
      Node: 16.14.2
      npm: 7.17.0
      Yarn: 1.22.19
      pnpm: 7.13.6
    Relevant packages:
      next: 13.1.0
      eslint-config-next: 13.1.0
      react: 18.2.0
      react-dom: 18.2.0

### Which example does this report relate to?

https://beta.nextjs.org/docs/installation

### What browser are you using? (if relevant)

Chrome 

### Describe the Bug

I just want to test Next 13.1, but it failed from start. 
On local, No CSS loaded, I tried to change to SCSS but still not work too
![image](https://user-images.githubusercontent.com/15695252/209207610-328e8bb2-7e06-4bd1-a2da-724c64d3226e.png)
On stackblitz I got. Project can't run
```
Attempted to load @next/swc-linux-x64-gnu, but an error occurred: Cannot load native addon because loading addons is disabled: /home/projects/nextjs-ktwu29/node_modules/@next/swc-linux-x64-gnu/next-swc.linux-x64-gnu.node
```

### Expected Behavior

CSS load normal

### To Reproduce

https://stackblitz.com/edit/nextjs-ktwu29?file=app/layout.tsx

**Comment by quick007** [comment]
Getting the same error locally. Glad it's not just me 😅

**Comment by quick007** [comment]
@IRediTOTO Can you change the name to be a bit more descriptive

**Comment by IRediTOTO** [comment]
Can you suggest me?

**Comment by Riko2909** [comment]
same here, but the css of some 3rd party UI-Libarys are loading!

**Comment by BWBama85** [comment]
Doesn't load tailwindcss after upgrade to 13.1.0 for me.

**Comment by AlainD-** [comment]
![nextjs-13 1 0-css-not-working-2](https://user-images.githubusercontent.com/5165564/209228497-b0457fb7-d307-4877-b227-ce7c215ac44c.png)

**Comment by kaceycleveland** [comment]
Has not worked since the past canary builds and 13.1.0; was hoping it would be fixed on 13.1.0.

**Comment by rkanani108** [comment]
Doesn't load tailwindcss after upgrade to 13.1.0 for me. Please guys look in to this.

**Comment by viperxeyes** [comment]
same here , css not loading in app directory only but works fine in pages.

**Comment by cpakken** [comment]
With the experimental appDir: true

`next dev --turbo` css works and loads fine
`next dev` css does NOT work

But `next build` still relies on the non turbo version so can't rely on turbo

**Comment by Usmani391** [comment]
Anyone got solution same with me tailwind is not loading

**Comment by huseyinkryl** [comment]
It happens to me with tailwindcss. I have tested 13.0.1

**Comment by AlexMcClay** [comment]
The same thing happens to me, no CSS is loading. With global CSS imported in the layout.tsx files and CSS modules in other .tsx files. I'm running windows 11.

**Comment by dhaaana** [comment]
The tailwindcss not loading issue also happens to me when I use the app directory. I've tried switching to the pages directory and the tailwindcss loads fine.

**Comment by viperxeyes** [comment]
If you have linux it works perfect , if you upload to vercel also works fine , the problem seems to only occure on windows through "dev and build" but not when uploaded to vercel or linux . someone need to look into this.

**Comment by Earle-Poole** [comment]
Adding onto the pile. 

Updated my project from 13.0.6 to 13.1 and Tailwind has stopped loading entirely, without any error messages on client or server.

**Comment by BWBama85** [comment]
> If you have linux it works perfect , if you upload to vercel also works fine , the problem seems to only occure on windows through "dev and build" but not when uploaded to vercel or linux . someone need to look into this.

I am beginning to think no one at Next.js actually tests the app folder on Windows before releasing a new version which is insane to me.

**Comment by nicolas-angelo** [comment]
Same issue for me on Windows. Just updated to 13.1.0 from the past canary version and Tailwind no longer works.
Works with turbo but I can't use turbo due to project setup

**Comment by AlainD-** [comment]
I have the same problem. I am on Windows, at the moment I `npm run dev — —turbo` but it raised other issues, not easy to work around. It’s a very short momentarily workaround but I really hope they will it very soon.

I wonder if it works in WSL2 in Windows though. Could be another temp solution in the meantime.

**Comment by selenecodes** [comment]
Having the same issue on Windows 10

**Comment by AlainD-** [comment]
There is a PR in progress for a fix: #44305

**Comment by timneutkens** [comment]
I've released a new stable patch version: https://github.com/vercel/next.js/releases/tag/v13.1.1, it'll take ~15mins before it's on npm. Happy holidays!
